canonical/juju-dashboard
# WD-8400 - Add update secrets panel

## Done

- Add update secrets panel.
- Remove expiry and rotate options as they are not support by user secrets.

## QA

- Go to the list of secrets.
- Open the action menu and click "Update".
- Change some values and save.
- Open the form again and you should see the new values.

## Details

https://warthogs.atlassian.net/browse/WD-8400
